xnat_tools/xnat2bids.py

```diff
@@ -34,13 +34,13 @@ def xnat2bids(
     bidsmap_file: str = typer.Option(
         "", "-f", "--bidsmap-file", help="Bidsmap JSON file to correct sequence names"
     ),
-    includeseq: List[int] = typer.Option(
+    includeseq: List[str] = typer.Option(
```

Comment:
It seems like typer will coerce it into a string even if someone passes an integer, correct? That would be very helpful for not breaking people's scripts.

Reply:
I'm not entirely sure.  At least for our integration / unit tests, Typer was throwing errors if an integer was passed rather than a string. I am interested to see the behavior once we deploy to Oscar.

Reply:
I'm fairly certain you're right.  I just tested with Typer, running "poetry run xnat2bids XNAT_E00114 ~/bids-expor
t -s 6" and no error was thrown.

Reply:
Same. That's great!